source/server.js

```diff
@@ -0,0 +1,192 @@
+/* Import express */
+const express = require('express');
+
+/* Create express server instance */
+const app = express();
+
+let scores;
+
+app.use(express.static('./source/frontend'));
+app.use(express.json());
+
+const fs = require('fs').promises;
+
+let turnCount = 0;
+
+const board = [
+  [0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0],
+];
+
+/* Reads from file and adds contents to global variable 'scores' */
+async function readScores() {
+  const rawData = await fs.readFile('./source/data/game.json', 'utf-8');
+  const parsedElement = JSON.parse(rawData);
```

Comment:
These two lines could be condensed to one
```suggestion
scores = JSON.parse(rawData);
```

Reply:
But having said that, this function would be easier to test and easier to understand if it returned the scores rather than setting a global variable. Everywhere the global variable is used, this function is called right before anyway, so there would be little impact on the rest of the code